File: server/static/go/v2/tools/genSession/session.go
package main

import (
	"encoding/base64"
	"encoding/json"
	"log"
)

type Session struct {
	UserID       int    `json:"userId"`
	UserName     string `json:"userName"`
	UserRealName string `json:"userRealName"`
	ClientIp     string `json:"clientIp"`
}

type ErpSession struct {
	PiId          int      `json:"piId"`
	EmployeeId    int      `json:"employeeId"`
	EmployeeName  string   `json:"employeeName"`
	DeptId        int      `json:"deptId"`
	DeptName      string   `json:"deptName"`
	CompId        int      `json:"compId"`
	CompName      string   `json:"compName"`
	EnglishName   string   `json:"englishName"`
	Phone         string   `json:"phone"`
	RoleCodeList  []string `json:"roleCodeList"`
	Type          int      `json:"type"`
	FilterIds     []int    `json:"filterIds"`
	FilterDeptIds []int    `json:"filterDeptIds"`
}

func main() {
	sess := &Session{UserID: 699}
	data, _ := json.Marshal(sess)
	log.Println("X-Session-Data", base64.StdEncoding.EncodeToString(data))
}

File: tools/serverless/src/ts/session.ts
interface ErpUserPara {
    piId?: string
    employeeId?: number
    employeeName?: string
    platformType?: number
    accountType?: number
    thirdCompId?: number
    deptId?: number
    deptName?: string
    compId?: number
    compName?: string
    englishName?: string
    phone?: string
    roleCodeList?: string[]
    type: number
    filterIds?: number[]
    filterDeptIds?: number[]
    filterCompIds?: number[]
}

interface Session {
    userId: number
    userName: string
    uerRealName: string
    clientIp: string
}

function generate() {
    const sess: ErpUserPara = {
        employeeId: 1,
        type: 1,
        platformType:101,
        thirdCompId:0,
        compId:10001,
        filterCompIds:[10001]
    }
    console.log(Buffer.from(JSON.stringify(sess)).toString('base64'))
}

const skipValidation = "e30="
generate()